cis194/week1/wei-ning/src/Exercise5.hs

```diff
@@ -0,0 +1,17 @@
+module Exercise5 (runTests) where
+
+import           Assert
+
+runTests :: IO ()
+runTests =
+  assert "ex1.5 expect total num of moves" $ 2^15 - 1 == length (hanoi 15 "a" "b" "c")
+
+type Peg = String
+type Move = (Peg, Peg)
+
+hanoi :: Integer -> Peg -> Peg -> Peg -> [Move]
+hanoi 1 from to _    = [(from, to)]
+hanoi n from to temp =
+  hanoi (n - 1) from temp to ++
```

Comment:
In Haskell, We usually align symbols. This means ++ will be at the beginning of a new line aligning with = sign above.

Reply:
will do. Thx